setup.cfg

```diff
@@ -126,6 +126,105 @@ images =
     accelerate~=0.23.0  # For the newer versions of Transformers
     pillow~=9.4.0
 
+heim =
+    # HEIM metrics
+    clip-anytorch~=2.5.0
+    NudeNet~=2.0.9
+    opencv-python~=4.7.0.68
+    Pillow~=9.4.0
+    pytorch-fid~=0.3.0
+    timm~=0.6.12
+    torch-fidelity~=0.3.0
+    torchmetrics~=0.11.1
+
+    accelerate~=0.23.0
+    antlr4-python3-runtime~=4.9.3
+    appnope~=0.1.3
+    asttokens~=2.2.1
+    astunparse~=1.6.3
+    autokeras~=1.0.20
+    backcall~=0.2.0
+    decorator~=5.1.1
+    diffusers~=0.13.1
+    einops~=0.6.0
+    executing~=1.2.0
+    flatbuffers~=1.12
+    flax~=0.6.3
+    ftfy~=6.1.1
+    gast~=0.4.0
+    google-auth-oauthlib~=0.4.6
+    google-cloud-core~=2.3.3
+    google-cloud-storage~=2.9.0
+    google-cloud-translate~=3.11.2
+    google-pasta~=0.2.0
+    grpcio~=1.51.1
+    grpcio-status~=1.48.2
+    h5py~=3.8.0
+    ipython~=8.9.0
+    jax~=0.3.25
+    jaxlib~=0.3.25
+    jedi~=0.18.2
+    keras~=2.9.0
+    Keras-Preprocessing~=1.1.2
+    keras-tuner~=1.2.0
+    kt-legacy~=1.0.4
+    libclang~=15.0.6.1
+    libcst~=0.4.9
+    lpips~=0.1.4
+    Markdown~=3.4.1
+    matplotlib-inline~=0.1.6
+    multilingual-clip~=1.0.10
+    oauthlib~=3.2.2
+    omegaconf~=2.3.0
+    opencv-python~=4.7.0.68
+    opt-einsum~=3.3.0
+    orbax==0.1.2
+    parso~=0.8.3
+    pexpect~=4.8.0
+    pickleshare~=0.7.5
+    prompt-toolkit~=3.0.36
+    proto-plus~=1.22.2
+    ptyprocess~=0.7.0
+    pure-eval~=0.2.2
+    pyDeprecate~=0.3.2
+    Pygments~=2.14.0
+    pytorch-fid~=0.3.0
+    pytorch-lightning~=2.0.5
+    requests-oauthlib~=1.3.1
+    stack-data~=0.6.2
+    SwissArmyTransformer~=0.2.1
+    tensorboard~=2.9.0
+    tensorboard-data-server~=0.6.1
+    tensorboard-plugin-wit~=1.8.1
+    tensorflow~=2.9.0
+    tensorflow-estimator~=2.9.0
+    tensorflow-io-gcs-filesystem~=0.30.0
+    termcolor~=2.2.0
+    timm~=0.6.12
+    torch-fidelity~=0.3.0
+    torchmetrics~=0.11.1
+    traitlets~=5.9.0
+    typing-inspect~=0.8.0
+    Unidecode~=1.3.6
+    wandb~=0.13.11
+    wcwidth~=0.2.6
+    Werkzeug~=2.2.2
+    wrapt~=1.14.1
+
+deep_floyd =
+    accelerate~=0.19.0
+    dacite~=1.6.0
+    diffusers[torch]~=0.16.1
+    pyhocon~=0.3.59
+    pymongo~=4.2.0
+    retrying~=1.3.3
+    safetensors~=0.3.1
+    sentencepiece~=0.1.97
+    sqlitedict~=1.7.0
+    tqdm~=4.64.1
+    transformers~=4.29.2
+    zstandard~=0.18.0
+
```

Comment:
Could you move this to its own `requirements-deep-floyd.txt` file? `setup.cfg` should be just for HELM stuff.

Reply:
I will move these to the deepfloyd script.